slasher/detection/proposals/detector.go

```diff
@@ -51,3 +51,26 @@ func (dd *ProposeDetector) DetectDoublePropose(
 	}
 	return nil, nil
 }
+
+// DetectDoubleProposeNoUpdate detects double proposals for a given block header by looking in the db.
```

Comment:
What does "NoUpdate" mean?

Reply:
Without db update that will result in effecting other proposals detection

Reply:
updated the comment . hope its ok now